package.json

```diff
@@ -65,6 +65,7 @@
     "react": "0.13.1",
     "react-hammerjs": "0.2.2",
     "react-imageloader": "1.2.0",
-    "xhr": "2.0.1"
+    "xhr": "2.0.1",
+    "git-rev-sync": "1.1.0"
```

Comment:
This can just be in `devDependencies` as it's not included in the build via `webpack`.